packages/gradle.properties

```diff
@@ -21,15 +21,8 @@ kotlin.mpp.stability.nowarn=true
 kotlin.code.style=official
 android.useAndroidX=true
 
-# We cannot enable hierarchical setup as common symbols are currently not visible in Android source
-# sets in consuming modules (https://youtrack.jetbrains.com/issue/KT-48153)
-#kotlin.mpp.enableGranularSourceSetsMetadata=true
-#kotlin.mpp.enableCompatibilityMetadataVariant=true
-#kotlin.native.enableDependencyPropagation=false
-
-# Hierachal builds are the default in Kotlin 1.6.20. Use old layout until https://github.com/realm/realm-kotlin/issues/1006
-# is fixed.
-kotlin.mpp.hierarchicalStructureSupport=false
+kotlin.mpp.hierarchicalStructureSupport=true
```

Comment:
Since we're using Kotlin `1.7.20` on master these flags are not necessary I think (they're  opt-in pre 1.6.20)

Reply:
But isn't it better to apply what we need explicitly? 🤔